test/routes/api/auth/auth.test.ts

```diff
@@ -54,17 +54,17 @@ test('POST /api/auth/login with invalid credentials', async (t) => {
 
   const testCases = [
     {
-      username: 'invalid_user',
+      email: 'invalid_email',
```

Comment:
This should be a 400 error, not 401.
You have to separate bad format email test from invalid email test (doesn't exist or doesn't match password).

Reply:
Should we really validate the email format during login attempts? While email validation is essential during registration, I think that it's redundant during login since the email format would have already been verified during the registration process.

Reply:
It is not necessary to work: if email is invalid, it will not be found in database.
But a badly formatted request is different from an illegitimate request.
To be fully compliant with the http spec, it's important to inform the client exactly what's going on.

Reply:
Ok no problem, I will wait for your PR (#68) to be merged to re-use your work on this part 👍